tools/msconvert/msconvert_macros.xml

```diff
@@ -25,7 +25,9 @@
     #else
       --input=${input}
       #if hasattr($input, 'display_name')
-      --input_name='${input.display_name}'
+      ##--input_name='${input.display_name}'
+      #set basename = $input.display_name
```

Comment:
@bgruening That's actually exactly what I originally did and it works fine (as long as --input_ident_name has the same prefix). I was trying to match the original logic as much as possible.

Incidentally, 20 out of 24 tests failed for this module (the updated one for this filter passed). Looks like mostly small differences due to changes in ontology version etc. I see that this tool is currently blacklisted in the build files, so I assume this is known and on the TODO list. I can try to address if time permits.